### PR TITLE
Modify SQL of getTypeDir

### PR DESCRIPTION
**Pull request by chenyang14**

### CODE REVIEW

**Comment by icecat2012** [comment]
@Billy4195 
I remember there was a commit rule a path format
`os.path.join(server_dir, "static/"+str(type_dir))`
as
`os.path.join(server_dir, "static", str(type_dir))`
right?

**Comment by Billy4195** [comment]
@icecat2012 the code you mention above will have the same result, and the second one is better, because in windows system the folder sign is `\` not `/`.

**Comment by icecat2012** [comment]
@chenyang14 
As Billy said, can you help us separate the route in os.path.join()?

**Comment by chenyang14** [comment]
Sure. I will modify them.

**Comment by Billy4195** [comment]
@chenyang14 Just change it in this PR. :+1:

File: server_api.py [hunk 1]
```diff
@@ -928,11 +928,11 @@ def upload_text_insert_db(json_obj):
...
         
         #get file place
-        sql = ("SELECT type_dir FROM data_type WHERE type_id=" + str(type_id))
-        pure_result = db.query(sql)
+        with DataTypeDao() as dataTypeDao:
+            type_dir = dataTypeDao.getTypeDir(typeId=str(type_id))
         try:
             text_system_name = text_id + ".txt"
-            system_file_dir = os.path.join(server_dir, "static/"+str(pure_result[0][0]))
+            system_file_dir = os.path.join(server_dir, "static/"+str(type_dir))
```
Comment (Billy4195) [inline server_api.py]: Change to `os.path.join(server_dir,"static",str(type_dir))`, or you can check whether the type of `type_dir` is str(), if yes then remove casting to str().

**Comment by chenyang14** [comment]
Finish modification

**Pull request merged**